### DISCUSSION TITLE
Some of mine

### DISCUSSION THREAD

**Discussion by jcc822** [ringdao-community-guild/discussion#35] [Q&A]
I have been following the development of Darwinia and also hold a token ring, but my understanding of your project is relatively vague. I only know it is a cross chain protocol and Evolution Land, but it lacks liquidity; In 2022, I was still watching the progress video introduction of your project on YouTube. Although the update was very slow, it is no longer available now. I can only check the update progress of the project through the media. I have not done a good job in popularizing science for the project, and there are few on the exchange, and there is basically no trading volume. Additionally, cooperation with other projects in the Polkadot ecosystem is also slow. I hope you can improve and make the project better!

**Comment by Cindy884** [comment]
Hi there! Thks for reaching out. 

**1.About DCDAO**

Darwinia Community DAO(DCDAO) is a self-organized community workgroup(WG) contributing to the growth of the Darwinia community. Darwinia Treasury sponsors it and Itering, donations from the community are also welcome.

Learn more about DCDAO at the following link. 👇 

[https://dcdao.notion.site/Darwinia-Community-DAO-1050ffdf6df34617bb90bb683ffc6bd5](https://www.notion.so/1050ffdf6df34617bb90bb683ffc6bd5?pvs=21)

If you have other questions, you can contact me via *cindy.tan@dariwiniacommunitydao.xyz* 

**2.About Darwinia**

2.1 **What is Darwinia?**

Darwinia is a cross-chain programmable network that provides two essential solutions: `Darwinia Chain` and `Darwinia Msgport`. 

`Darwinia Chain` is a smart contract platform compatible with the EVM and shares the security of Polkadot. On the other hand, `Darwinia Msgport` acts as a crucial infrastructure enabling Dapps to seamlessly communicate and exchange messages across different blockchain networks, facilitating efficient interchain operations.

**2.2 Darwinia Roadmap**

Check the following link. 👇

[https://darwinia.notion.site/a76d5bdc7ad242bea991c023dfca208f?v=5fb127fc9a7a4d67a3bbc3ed50445180](https://www.notion.so/a76d5bdc7ad242bea991c023dfca208f?pvs=21)

**2.3 Q&A**

Check the following link. 👇

[https://darwinia.notion.site/a76d5bdc7ad242bea991c023dfca208f?v=5fb127fc9a7a4d67a3bbc3ed50445180](https://www.notion.so/a76d5bdc7ad242bea991c023dfca208f?pvs=21) 

If you wanna get technical support from the Darwinia team, you can contact them via e-mail: *hello@darwinia.network*

**Comment by wuminzhe** [comment]
@jcc822 you are right. 

I think one of the reasons is darwinia's lack of education to the community. The another reason is that the darwinia changes as the crypto industry changes. [Darwinia is now more than a product but a brand and dao.](https://darwinia.notion.site/What-is-Darwinia-c3e845a9a141425abad0a4c92c993d38?pvs=4)

and, can you tell more about what knowledge you are interested in? It's very helpful. We are putting more efforts in education.
